torch/csrc/jit/fuser/cuda/resource_strings.h

```diff
@@ -125,10 +135,8 @@ constexpr auto rand_support_literal = R"(
     static const unsigned long kPhiloxSB = 0xCD9E8D57;
   };
 
-  // Inverse of 2^32.
-  #define M_RAN_INVM32 2.3283064e-10f
   __device__  __inline__ float uniform(unsigned int x) {
-    return x * M_RAN_INVM32;
+    return (x & ((1ULL << 24) - 1)) * ::ldexp(1.0, -24);
```

Comment:
Same comment as above, I'm not convinced that throwing our random bits is the way to go.

Reply:
Keeping this method since I am not sure if dividing by 2^32 and filtering out 1s would remove bias.